strategy/plan2020.md

```diff
@@ -0,0 +1,208 @@
+   - Update tutorial (live and self-paced) information based on current best practices
+   - Establish a system for regularly revisiting documentation to keep it up-to-date
+2. CF-Xarray & GeoCAT Development / Decommission ESMLab [5+1 points]
+   - New blog post? (about the plan/initiative) Or some other discussion/announcement
+     forum?
+   - Triage existing features of ESMLab
+     - Does this functionality already exist somewhere else?
+       CF-Xarray or GeoCAT or elsewhere
+     - Where should they move?  CF-Xarray or GeoCAT or elsewhere
+     - Create matching issue in appropriate location
+   - Triage existing issues in ESMLab
+     - To where should the feature these issues pertain move?
+       CF-Xarray or GeoCAT or elsewhere
+     - Move (or re-write with attribution) issues in appropriate new location
+   - Determine a priority order for new issues
+   - Schedule a series of mini hackathons to address each issue in priority order
+   - Once ESMLab-related issues are complete, decommission ESMLab
+   - New blog post?
+   - Determine if it is valuable to continue these hackathons specifically to
+     continue contributions to either CF-Xarray, GeoCAT, or another repository
+3. Conda Metapackage [5 points]
+   - Identify necessary base packages for typical Pangeo environment
+     (e.g., from the tutorial)
+     - Can we have Conda packages install labextensions, if necessary?
+   - Create a suite of unit tests that can be used to validate if the
+     environment is working (i.e., common problems, routine operations, etc.)
+   - Create a repository that implements CI to:
+     - Install the environment with the most recent versions of environment packages
+     - Detect if any versions of packages have changed
+     - If new versions are detected, run the test suite
+     - If (and only if) tests pass, mint a new version of the environment
+     - If a new version of the environment is created, publish a new version
+       of the metapackage
+     - Implement this workflow periodically (e.g., every week, or every day)
+   - Use the repository + CI to deliver regular updates to a Conda metapackage
+   - New blog post!
+4. Cloud Deployment Scoping [5 points]
+   - Scope out a use case for learning how to deploy and maintain a cloud
+     Pangeo hub of our own
+   - Some ideas to explore:
+     - Determine how to obtain the cheapest deployment possible for various purposes:
+       education/training, science/research, development/testing
+     - A specialized hub for online tutorials/self-paced learning
+     - A small-team research deployment (e.g., for Matt Long’s group, or for us
+       to use to test notebooks and notebook scaling solutions)
+     - Target a new service to implement that would need a testing/development hub
+       (e.g., a notebook sharing/publication repository)
+   - Determine best use cases (1-2) and deploy hubs for those use cases
+   - Blog post
+5. Xdev Communication Strategy [4+1 points]
```

Comment:
Is "4+1 points" different from 5? Not sure how to read that.

Reply:
Yeah.  I try to explain that in the next section, but that's bad practice on my part.  I should explain it in the intro text at the top of the section.

Reply:
...Will fix.

Reply:
@jukent Take a look at it, now.

Reply:
Makes sense!